NuimoSDK/Properties/AssemblyInfo.cs

```diff
@@ -1,18 +1,20 @@
 ﻿using System.Resources;
 using System.Reflection;
+using System.Runtime.CompilerServices;
 using System.Runtime.InteropServices;
 
 // General Information about an assembly is controlled through the following 
 // set of attributes. Change these attribute values to modify the information
 // associated with an assembly.
-[assembly: AssemblyTitle("Nuimo SDK")]
-[assembly: AssemblyDescription("Library for Windows 10 platforms to connect and communicate with Nuimo controllers made by Senic")]
+[assembly: AssemblyTitle("NuimoSDK")]
+[assembly: AssemblyDescription("")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Senic GmbH")]
-[assembly: AssemblyProduct("Nuimo SDK")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("NuimoSDK")]
```

Comment:
Is it necessary to rename the product? Does your change conform to a naming convention?